scripts/apps/web-publisher/controllers/WebPublisherManagerController.js

```diff
@@ -260,14 +275,105 @@ export function WebPublisherManagerController($scope, publisher, modal) {
 
         /**
          * @ngdoc method
+         * @name WebPublisherManagerController#toogleCreateListCard
+         * @param {String} listType - type of content list
+         * @description Creates a new unsaved content list card
```

Comment:
The description of this method gives no indication as to what is being toggled. Can we maybe clarify?
Is the description correct? Should we change it? If not, should we change the method name?

By looking at the implementation, I don't see anything being toggled. So maybe remove the `toggle` prefix?